<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExWidget</class>
 <widget class="QWidget" name="ExWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExWidget</string>
  </property>
  <widget class="QPlainTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>399</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>122</width>
     <height>325</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="0">
     <widget class="QPushButton" name="btnMetaObject">
      <property name="text">
       <string>类的元对象信息</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="labGril">
      <property name="text">
       <string>设置女生年龄</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QSpinBox" name="spinGril">
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QPushButton" name="btnClean">
      <property name="text">
       <string>清空文本框</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btnBoyAdd">
      <property name="text">
       <string>Boy:Age 增加</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="btnGrilAdd">
      <property name="text">
       <string>Girl:Age 增加</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSpinBox" name="spinBoy">
      <property name="value">
       <number>20</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labBoy">
      <property name="text">
       <string>设置男生的年龄：</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
